app/src/main/java/app/gamenative/utils/LsfgVkManager.kt

```
@@ -116,6 +131,108 @@ object LsfgVkManager {
+                    val statsFile = File(container.rootDir, STATS_RELATIVE_PATH)
+                    if (statsFile.isFile &&
+                        System.currentTimeMillis() - statsFile.lastModified() <= STATS_FRESHNESS_MS
+                    ) {
+                        statsFile.readLines()
+                            .firstOrNull { it.startsWith("fps=") }
+                            ?.substringAfter("fps=")
+                            ?.toFloatOrNull()
+                    } else {
+                        null
+                    }
+                } catch (t: Throwable) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)